src/main/scala/util/CoreMonitor.scala

```diff
@@ -22,7 +22,12 @@ class CoreMonitorBundle(val xLen: Int) extends Bundle with Clocked {
   val inst = UInt(width = 32.W)
 }
 
+class CSRMonitorBundle(val xLen: Int) extends Bundle with Clocked {
+  val mie = UInt(width = xLen.W)
```

Comment:
is there any reason you've picked  these two? Why not mstatus, etc?

Reply:
Those two are the one that @surajvellengar wanted